File: twiml/voice/application/reject-parameter/reject-parameter.6.x.php
<?php
require_once './vendor/autoload.php';
use Twilio\TwiML\VoiceResponse;

$response = new VoiceResponse();
$reject = $response->reject();
$reject->parameter(['name' => 'reject_reason', 'value' => 'no agents available']);

echo $response;

File: twiml/voice/application/reject-parameter/reject-parameter.6.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var reject = new Reject();
        reject.Parameter(name: "reject_reason", value: "no agents available");
        response.Append(reject);

        Console.WriteLine(response.ToString());
    }
}
